brokernode-api/service/BrokersService.js

```diff
@@ -0,0 +1,148 @@
+'use strict';
+
+
+/**
+ * Inform broker that the webnode has done
+ *
+ * id Long Node id
+ * body List List of trytes
+ * returns List
+ **/
+exports.informBroker = function(id,body) {
```

Comment:
We should change "informBroker" to "informBrokerBrokerAddress" if we're going to call the other method "informBrokerGenesisHash"

Reply:
this is generated file from swagger, we will not use it in future..